extensions/amp-mega-menu/0.1/test/test-amp-mega-menu.js

```diff
@@ -82,7 +82,42 @@ describes.realWin(
       const headingClass = '.i-amphtml-mega-menu-heading';
       expect(element.querySelectorAll(headingClass).length).to.equal(3);
       const contentClass = '.i-amphtml-mega-menu-content';
-      expect(element.querySelectorAll(contentClass).length).to.equal(3);
+      expect(element.querySelectorAll(contentClass).length).to.equal(2);
+    });
+
+    it('should add event listeners on component layout', async () => {
+      await element.build();
+      const impl = element.implementation_;
+      const heading = doc.getElementById('heading1');
+      const listenersSpy = sandbox.spy(impl, 'registerMenuItemListeners_');
+      await element.layoutCallback();
+      expect(listenersSpy).to.be.calledOnce;
+      const clickEvent = new Event('click');
+      const headingClickSpy = sandbox.spy(impl, 'shouldHandleClick_');
```

Comment:
Here I want to ideally spy on either `headingClickHandler_` or `handleHeadingClick_` directly, but neither seems to work (probably because of `.bind()` which creates a new function?)